_posts/newsletters/2020-04-13-newsletter.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: US-RSE Newsletter
+subtitle: April 2020 Newsletter
+category: newsletter
+tags: [newsletter, workshops, jobs]
+---
+
+In this bi-monthly newsletter, we share recent, current, and planned activities of the US-RSE Association. Newsletters are also available on our [website](https://us-rse.org/newsletters/) alongside the growing resources and information on the US-RSE Association. A sign-up option for our newsletter is available [here](https://us-rse.org/join/).
+
+In this issue:
+
+*   [US-RSE Virtual Workshop April 22-23](#virtual)
+*   [US-RSE Association Workshop at Princeton moving to October](#princeton)
+*   [Workshops at PEARC20 and SC20](#workshops)
+*   [Recent Job Postings](#jobs)
+*   [Interesting Reads](#reads)
+*   [Get Involved](#getinvolved)
+
+<a name="virtual"></a>
+# US-RSE Virtual Workshop
+
+We are excited to announce that the US-RSE association will host a virtual workshop **on April 22 and 23!** This workshop will be held in three online sessions:
+
+1.  The RSE Landscape - Reports on RSE groups and activities, with discussion (Wednesday 4/22, 12:00-1:30 PM EDT)
```

Comment:
Feedback here is the same that I gave to @cosden - I think this reads much more easily when the title portion (e.g., "The RSE Lanscape" is bolded.